### PR TITLE
Feature.mgreene disconnected service

### PR DESCRIPTION
**Pull request by zancas**
@mattgreene 
#### What issues does this address?
Fixes #180 

DON'T MERGE YET.   DOCUMENTATION NECESSARY.

many flake8 violations remain.

### CODE REVIEW

**Comment by mattgreene** [comment]
Please generate an updated PR showing just the diff from adding your new tests.  This PR shows everything going back to my implementation of the feature.  I can't tell what is just your new code that needs review.

File: test/functional/neutronless/test_disconnected_service.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    monitor['provisioning_status'] = 'PENDING_CREATE'
+for member in CONNECTED_SVC['members']:
+    member['provisioning_status'] = 'PENDING_CREATE'
+    member['network_id'] = u'a8f301b2-b7b9-404a-a746-53c442fa1a4f'
+
+
+@pytest.fixture(scope='module')
+def setup_registry_snapshot(bigip):
+    # Setup device registries
+    pp('inside setup_registry_snapshot')
```
Comment (mattgreene) [inline test/functional/neutronless/test_disconnected_service.py]: Remove debugging output.

File: test/functional/neutronless/test_disconnected_service.py [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    return frozenset(register_device(bigip))
+
+
+@pytest.fixture
+def setup_neutronless(request, bigip, setup_registry_snapshot):
```
Comment (mattgreene) [inline test/functional/neutronless/test_disconnected_service.py]: Move this fixture to a common place for all neutronless functional tests to share.

File: test/functional/neutronless/test_disconnected_service.py [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+        ordered = _deletion_order(created)
+        for selfLink in ordered:
+            if 'virtual-address' not in selfLink:
+                posttest_registry[selfLink].delete()
+
+    request.addfinalizer(remove_test_created_elements)
+    return bigip, wrappedicontroldriver, pretest_snapshot
+
+
+def test_disconnected_service(setup_neutronless):
```
Comment (mattgreene) [inline test/functional/neutronless/test_disconnected_service.py]: This replaces the functional test I wrote that requires a live deployment.  There are many additional tests required to qualify the feature.

- create/delete of the 'disconnected network' (i.e. ppp interface) upon tenant folder creation/deletion
- 3 new ini settings (at least the two for polling interval and gross timeout)
- "edge" tests such as a service containing the seg_id immediately upon creation

**Comment by zancas** [comment]
This PR is superseded by #184.

**Pull request closed without merging**